src/Aspire.ProjectTemplates/templates/aspire-py-starter/.template.config/template.json

```diff
@@ -0,0 +1,79 @@
+{
+  "$schema": "https://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [
+    "Common",
+    "Aspire",
+    "Cloud",
+    "Web",
+    "Web API",
+    "API",
+    "Service",
+    "Python",
+    "JavaScript"
+  ],
+  "name": "Aspire Python Starter App",
+  "defaultName": "AspirePyApp",
+  "description": "A project template for creating an Aspire app with a Python backend API service and a JavaScript frontend using Vite and React.",
+  "shortName": "aspire-py-starter",
+  "sourceName": "AspirePyApplication.1",
+  "preferNameDirectory": true,
+  "tags": {
+    "language": "C#",
+    "type": "solution",
+    "editorTreatAs": "solution",
```

Comment:
Hmmm not sure if this is the right choice given this is using file-based app host and no projects or laying down a solution.